index.html

```diff
@@ -0,0 +1,2199 @@
+              <div class="d-flex align-items-center font-roboto signup-call">
+                <span class="logo-icon-newsletter">
+                  <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-icon">
+                </span>
+                  <h2 class="newsletter-title">
+                    SIGN UP FOR OUR NEWSLETTER
+                  </h2>
+              </div>
+                <div>
+                  <button type="button" class="btn sign-up-btn">Sign Up ></button>
+                </div>
+              <div>
+                <a class="font-roboto pu-link" href="#">Update your preferences»</a>
+              </div>
+            </div>
+          </section>
+
+          <div class="d-none d-lg-block">
+            <article class="card rounded-0">
+              <div>
+                <img class="rounded-0 w-100 border-0 border-top-0" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="President Trump">
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
+              </div>
+              <div class="card-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-3">
+                    COVID-19 Linked to Changes to the Brains of Some Patients
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Have the pollsters learned from their mistakes in the 
+                  2016 presidential election? Maybe. Maybe not
+                </p>
+              </div>
+            </article>
+          </div>
+
+          <div class="d-none d-lg-block">
+            <article class="card rounded-0">
+              <div>
+                <img class="rounded-0 w-100 border-0 border-top-0" src="https://d.newsweek.com/en/full/1581097/trump-flag-pollsters-2016-2020.jpg" alt="President Trump">
+                <a href="#">
+                  <p class="politics">POLITICS</p>
+                </a>
+              </div>
+              <div class="card-body">
+                <a href="#">
+                  <h2 class="mag_link2 article-title pt-3">
+                    COVID-19 Linked to Changes to the Brains of Some Patients
+                  </h2>
+                </a>
+                <p class="card-text">
+                  Have the pollsters learned from their mistakes in the 
+                  2016 presidential election? Maybe. Maybe not
+                </p>
+              </div>
+            </article>
+          </div>
+         
+        </div>
+      </div>
+
+      <div class="row second_section_wrapper pb-5 border-bottom">
+        <section class="mag-wrapper">
+          <div class="d-flex align-items-center justify-content-center border-bottom font-roboto w-100 py-3 mag-heading w-100">
+            <img class="newsweek-icon" src="./new_icon.PNG" alt="newsweek-logo">
+            <h2 class=" mag-title px-2">IN THE MAGAZINE</h2>
+          </div>
+
```

Comment:
The images of this section have a different size than the original page. Please fix them.
Yours:
![Screenshot from 2020-04-22 13-18-09](https://user-images.githubusercontent.com/51926937/80018577-cf3e8600-849b-11ea-9ba2-abb452207bde.png)

Original:
![Screenshot from 2020-04-22 13-18-02](https://user-images.githubusercontent.com/51926937/80018588-d5ccfd80-849b-11ea-85d9-c3d66f364334.png)

The "see all features" button is on top of the image when you make the screen smaller. That shouldn't happen.